src/oneprovision/lib/provision.rb

```diff
@@ -163,7 +184,8 @@ def status
         # @param config  [String]  Path to the configuration file
         # @param cleanup [Boolean] True to delete running VMs and images
         # @param timeout [Integer] Timeout for deleting running VMs
-        def create(config, cleanup, timeout)
+        # @param skip    [Symbol]  Skip provision, config or none phases
+        def create(config, cleanup, timeout, skip)
```

Comment:
Please update this method so there is just single create action for each entity and method **ends at its end**, not in the middle by 2 extra checks/creates and returns.